app.rb

```diff
@@ -148,6 +148,7 @@ def authorization_url(ial:, aal: nil)
         state: random_value,
         nonce: random_value,
         prompt: 'select_account',
+        irs_attempts_api_session_id: random_value,
```

Comment:
if we wanted to ship this and use it in deployed sample apps, we could add an "enable attempts api" checkbox in the options dropdown to toggle this behavior on/off